invenio_migrator/legacy/records.py

```diff
@@ -110,7 +110,7 @@ def dump(recid, from_date, with_json=True, latest_only=False, **kwargs):
         revision_iter = reversed(get_record_revisions(recid))
 
     # Dump revisions
-    record_dump = dict(record=[], files=[])
+    record_dump = dict(record=[], files=[], recid=recid)
```

Comment:
if we add the `recid` here, probably we can remove from https://github.com/inveniosoftware/invenio-migrator/pull/5/files#diff-e2f56d0aea32d4543cbd4eb624c94517R122